sklearn/metrics/_plot/tests/test_confusion_matrix_display.py

```diff
@@ -0,0 +1,382 @@
+from numpy.testing import (
+    assert_allclose,
+    assert_array_equal,
+)
+import numpy as np
+import pytest
+
+from sklearn.datasets import make_classification
+from sklearn.compose import make_column_transformer
+from sklearn.exceptions import NotFittedError
+from sklearn.linear_model import LogisticRegression
+from sklearn.pipeline import make_pipeline
+from sklearn.preprocessing import StandardScaler
+from sklearn.svm import SVC
+from sklearn.svm import SVR
+
+from sklearn.metrics import ConfusionMatrixDisplay
+from sklearn.metrics import confusion_matrix
+
+
+# TODO: Remove when https://github.com/numpy/numpy/issues/14397 is resolved
+pytestmark = pytest.mark.filterwarnings(
+    "ignore:In future, it will be an error for 'np.bool_':DeprecationWarning:"
+    "matplotlib.*"
+)
```

Comment:
should this be something in `fixes.py`?

Reply:
I am not sure. We would need an import instead of creating the filter for the entire test file.
IMO, it seems more explicit in this way.